climt/_core/initialization.py

```diff
@@ -428,11 +454,11 @@ def init_interface_level_tau_longwave(array_dims, quantity_description, initial_
         'default_value': 'sea',
         'dtype': 'a100'
     },
-    'snow_ice_temperature': {
+    'snow_ice_temperature_poly': {
```

Comment:
Is snow_ice_temperature_poly the best name for this quantity? If the standard name is "snow ice" instead of just "snow", then that's fine, though I'd think "snow_temperature" is clear enough. As it is, I have no idea what the "_poly" means - it should probably be expanded into the full word(s).

Reply:
This is a composite "slab" of snow and ice -- hence the name. You need to solve the heat equation for a surface that is either snow or snow+ice (land or sea), and it is much easier to solve them as one entity (else there will be inconsistencies at the boundary)

Reply:
will change the name, though -- snow_ice_spline might be more apt.

Reply:
This is not very clear by reading the name (it reads as "snow ice" is just "snow", a type of ice). Would it be better use snow_and_ice_ or snow_and_ice_composite_ or snow_ice_composite_ or snow_and_ice_slab_, or frozen_water_, or just ice_ (since snow is ice)?

Reply:
ok. Then, I would go with something like snow_and_ice_temperature. Think it conveys the information without being confusing (slab of snow? composite what?).

Reply:
Sounds good to me.